.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release to Maven Central
+
+on:
+  workflow_dispatch:
+  release:
+
+jobs:
+  release:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v4
+
+        - name: Set up JDK 17
+          uses: actions/setup-java@v4
+          with:
+            java-version: '17'
+            distribution: 'temurin'
+
+        - name: Setup Gradle
+          uses: gradle/actions/setup-gradle@v3
+
+        - name: Release
+          env:
+            AUTOMATED_MAVEN_RELEASE_PGP_SECRET: ${{ secrets.AUTOMATED_MAVEN_RELEASE_PGP_SECRET }}
+            AUTOMATED_MAVEN_RELEASE_GITHUB_APP_PRIVATE_KEY: ${{ secrets.AUTOMATED_MAVEN_RELEASE_GITHUB_APP_PRIVATE_KEY }}
+            AUTOMATED_MAVEN_RELEASE_SONATYPE_PASSWORD: ${{ secrets.AUTOMATED_MAVEN_RELEASE_SONATYPE_PASSWORD }}
+# These don't exist and need to be provided somehow.
+            GPG_KEY_PASSWORD: ${{ secrets.GPG_PASSWORD }}
```

Comment:
Note that `AUTOMATED_MAVEN_RELEASE_PGP_SECRET` [as defined for `gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/docs/credentials/generating-credentials.md#generating-a-new-pgp-key) is a [passphrase-less private key](https://unix.stackexchange.com/a/550538/46453), so there is no password for this value - does the gradle build definitely require a password? Will it accept a blank value?